__tests__/components/ErrorBoundary.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { it, expect, describe, afterEach } from 'vitest';
+import { cleanup, render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/vitest';
+import ErrorBoundary from '../../src/components/ErrorBoundary';
```

Comment:
It might cause warning messages because of missing the `"jsx": "react-jsx"` in the compilerOptions for vitest